src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import "./Todo.css";
+
+function Todo(props) {
+    return (
+        <div className="todo-item">
+            <input type='checkbox' className="checkbox" />
+            <p>{props.task.task}</p>
```

Comment:
you don't have to do props.task.task. The reason being because when you define what task is, it's only going to care about the last item in the object. In this case it will be task, you can simply write props.task.